test/edu-benefits/1995/00-required.e2e.spec.js

```diff
@@ -0,0 +1,102 @@
+if (process.env.BUILDTYPE !== 'production') {
+  const E2eHelpers = require('../../util/e2e-helpers');
+  const Timeouts = require('../../util/timeouts.js');
+  const EduHelpers = require('../../util/edu-1995-helpers');
+
+  module.exports = E2eHelpers.createE2eTest(
+    (client) => {
+      EduHelpers.initApplicationSubmitMock();
+
+      // Ensure introduction page renders.
+      client
+        .url(`${E2eHelpers.baseUrl}/education/apply-for-education-benefits/application/1995`)
+        .waitForElementVisible('body', Timeouts.normal)
+        .assert.title('Apply for education benefits: Vets.gov')
+        .waitForElementVisible('div.form-progress-buttons', Timeouts.slow)
+        .click('.form-progress-buttons .usa-button-primary');
```

Comment:
I think I'm going to leave this for now. My suggestion doesn't really seem clearer and I don't think we should add a class just for testing if we can avoid id.